src/manifest/mod.rs

```diff
@@ -169,7 +148,53 @@ impl CargoManifest {
             files: files,
             main: js_file,
             types: dts_file,
+        })
+    }
+
+    fn into_esmodules(mut self, scope: &Option<String>, disable_dts: bool) -> NpmPackage {
+        let filename = self.package.name.replace("-", "_");
+        let wasm_file = format!("{}_bg.wasm", filename);
+        let js_file = format!("{}.js", filename);
+
+        let dts_file = if disable_dts == true {
+            None
+        } else {
+            Some(format!("{}.d.ts", filename))
+        };
+
+        if let Some(s) = scope {
+            self.package.name = format!("@{}/{}", s, self.package.name);
+        }
+        let mut files = vec![wasm_file, js_file.clone()];
```

Comment:
this `clone`. i hate it but i also love not having a ton of type annontations everywhere. i think it's fine but if we hate it let's talk about ways to make it as clean as possible.

Reply:
I don't think this single allocation that isn't in a loop (or even within any code that we know to be hot) is a problem. :+1: